README.md

```diff
@@ -0,0 +1,20 @@
+<img alt="Netlify Status"
+     src="https://api.netlify.com/api/v1/badges/269f7467-6afd-49ae-97f2-61a160e93a9a/deploy-status">
+<img alt="CI"
+     src="https://github.com/nix-dot-dev/nix.dev/workflows/CI/badge.svg">
```

Comment:
@domenkozar: do you want this image to link to somewhere?

Reply:
nah :)